package/yast2-storage.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Apr  6 15:16:08 UTC 2022 - José Iván López González <jlopez@suse.com>
+
+- Partitioner: PVs are not wrongly removed when resizing a VG
```

Comment:
:thinking: Not sure if I got it. Not wrongly removed?

Reply:
Yes, one bad thing happening in the bug report is that some PVs are removed, but they shouldn't.

Reply:
Ah! Sorry, I got it now. You mean that the change is not removing PVs by mistake. Thanks!